doc/cli/operator-sdk_cleanup.md

```diff
@@ -13,15 +13,16 @@ operator-sdk cleanup [flags]
 ### Options
 
 ```
-      --kubeconfig string         The file path to kubernetes configuration file. Defaults to location specified by $KUBECONFIG, or to default file rules if not set
-      --namespace string          The namespace where the operator watches for changes.
-      --olm                       The operator to be cleaned up is managed by OLM in a cluster. Cannot be set with another cleanup-type flag (default true)
-      --manifests string          [olm only] Directory containing package manifest and operator bundles.
-      --operator-version string   [olm only] Version of operator to deploy
-      --install-mode string       [olm only] InstallMode to create OperatorGroup with. Format: InstallModeType=[ns1,ns2[, ...]]
-      --include strings           [olm only] Path to Kubernetes resource manifests, ex. Role, Subscription. These supplement or override defaults generated by run/cleanup
-      --timeout duration          [olm only] Time to wait for the command to complete before failing (default 2m0s)
-  -h, --help                      help for cleanup
+      --kubeconfig string           The file path to kubernetes configuration file. Defaults to location specified by $KUBECONFIG, or to default file rules if not set
+      --namespace string            The namespace where the operator watches for changes.
```

Comment:
```suggestion
      --namespace string            (Deprecated) - The namespace where the operator watches for changes.
```

Reply:
Did you not change this as well?

Reply:
I think we need to change all to keep the same behaviour across the project. Otherwise is hard to know when use  1 or another. Is it make sense?

Reply:
done